here-naksha-lib-psql/src/commonMain/kotlin/naksha/psql/executors/query/WhereClauseBuilder.kt

```diff
@@ -160,6 +161,80 @@ class WhereClauseBuilder(private val request: ReadFeatures) {
         }
     }
 
+    private fun whereTags() {
+        request.query.tags?.let { tagQuery ->
```

Comment:
why .let ? Maybe I'm old-fashioned, but it looks a little weird, as I understand it replaces `if(tags != null)`, but I had to read it twice to understand :)

Reply:
you are totally right, this is not clear - will change